### PR TITLE
Nodejs week1

### PR DESCRIPTION
**Pull request by danJecu**
Homework for NodeJS Week 1

### CODE REVIEW

File: src/backend/app.js [hunk 1]
```diff
@@ -19,16 +19,104 @@ app.use(express.json());
...
+  try {
+    const futureMeals = await knex.raw(
+      'SELECT * FROM meals WHERE created_date >= NOW()'
+    );
+
+    // I selected the first item in the array to avoid the _buf object
+    if (!futureMeals[0].length) {
+      response
+        .status(404)
+        .json({ error: 'There are no meals created after this date.' });
```
Comment (Divyajg) [inline src/backend/app.js:35]: Even though there are no meals available, it shouldn't be an error response as it is responding with the correct data.
So the response is `Status:500` and` result:"There are not meals available."`.

File: src/backend/app.js [hunk 2]
```diff
@@ -19,16 +19,104 @@ app.use(express.json());
...
+    } else {
+      response.json(futureMeals[0]);
+    }
+  } catch (err) {
+    response.status(500).json({ error: err.message });
```
Comment (Divyajg) [inline src/backend/app.js:40]: The status response at this part is an error. So `Status:404` and `result:error`.

**Review by Divyajg — CHANGES_REQUESTED** [review]
Everything seems good. But there seems to be a confusion in defining status response. I hope you can understand the explanation I have given. 
Text me in slack, if you need any help.
For now correct the changes in the responses.

File: src/backend/app.js (hunk 1 above)
Comment (danJecu) [reply src/backend/app.js:35]: It was fixed

**Pull request merged**